docs/integration-services/lesson-1-create-a-project-and-basic-package-with-ssis.md

```diff
@@ -53,7 +54,7 @@ Here is an example of the source data in the SampleCurrencyData.txt file:
 1.00070049USD9/9/05 0:001.001502253  
 1.00070049USD9/10/05 0:000.99990001  
 1.00020004USD9/11/05 0:001.001101211  
-1.00020004USD9/12/05 0:000.99970009</pre>  
+1.00020004USD9/12/05 0:000.99970009```

```

Comment:
Please put the tick marks on its own line after this code block.

Reply:
I can apply these fixes . . .